src/TetherComponent.jsx

```diff
@@ -75,18 +78,43 @@ class TetherComponent extends Component {
   componentWillUnmount() {
     this._destroy()
   }
-
+  
+  getTether() {
+    return this._tether
+  }
+  
   disable() {
     this._tether.disable()
   }
 
   enable() {
     this._tether.enable()
   }
+  
+  on(event, handler, ctx) {
+    this._tether.on(event,handler,ctx);
+  }
+  
+  once(event, handler, ctx) {
+    this._tether.once(event,handler,ctx);
+  }
+  
+  off(event, handler) {
+    this._tether.off(event,handler)
+  }
 
   position() {
     this._tether.position()
   }
+  
+  _registerEventListeners() {
+    if ( this.props.onUpdate ) {
```

Comment:
another nitpick, but can we please get the spaces inside the parens removed from here as well as the other to match the original code style.